src/components/Welcome.js

```diff
@@ -30,20 +32,34 @@ const Welcome = ({ setToken }) => {
   return (
     <div>
       <h1>Welcome to your Smart Shopping list!</h1>
-      <Link to="/list" onClick={setTokenStorage}>
+      <Button
+        component={Link}
+        variant="contained"
+        color="secondary"
+        to="/list"
+        onClick={setTokenStorage}
+      >
         Create New List
-      </Link>
+      </Button>
       <p>- or -</p>
       <p>Join an existing shopping list by entering a three word token.</p>
       <form onSubmit={handleSubmit}>
         <div>
           <label htmlFor="join-list">Enter a share token</label>
         </div>
-        <input
+        <TextField
```

Comment:
If you want the Button and TextField to be more in alignment, wrap them in a div and set:
`display: flex, justifyContent: center`
![image](https://user-images.githubusercontent.com/10378331/94262784-9f3b5900-fef9-11ea-8a5f-bf4c55466261.png)